src/frontend/src/pages/HomePage/UsefulLinks.tsx

```diff
@@ -4,52 +4,49 @@
  *
  */
 
+import { useTheme } from '@mui/material';
 import {
-  faCoins,
-  faReceipt,
-  faHandHoldingUsd,
-  faDollarSign,
-  faFileAlt,
-  faSortNumericDown,
-  faCogs
-} from '@fortawesome/free-solid-svg-icons';
+  ShoppingCart,
+  Settings,
+  Receipt,
+  Description,
+  CurrencyExchange,
+  AttachMoney
+} from '@mui/icons-material';
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
 import { Grid } from '@mui/material';
```

Comment:
these import warnings are from resolving the merge conflict, can u just remove the unused imports, that's my b

Reply:
box and externalLink are unused, forgot the warnings don't show up in the conversation part of github